sql-cli/sql_cli/configuration.py

```diff
@@ -19,18 +19,20 @@ class Config:
     """
 
     project_dir: Path
-    environment: str
+    environment: str | None = None
 
     connections: list[dict[str, Connection]] = field(default_factory=list)
     airflow_home: str | None = None
     airflow_dags_folder: str | None = None
 
-    def get_env_config_filepath(self) -> Path:
+    def get_env_config_filepath(self) -> Path | None:
         """
         Return environment specific configuration.yaml filepath.
 
         :returns: The path to the desired YAML configuration file
         """
+        if not self.environment:
+            return None
```

Comment:
This doesn't looks right to me. Why do we call this function if we don't need the env config filepath?

Reply:
Since we're allowing Config to be created without setting environment, we will need to take care of it in this method right? I mean if someone calls this method after they have created a Config with None environment.